docs/knowledge_base/tips/the_data_nodes/index.md

```diff
@@ -0,0 +1,151 @@
+
+Taipy is a Python tool for making web applications that use data. 
+It can do many things, but we're going to talk about 
+two important things in Taipy: Data nodes and Tasks.
+
+Data nodes are like a bridge to get data from different places. 
+They help us access data easily. This article is mostly about data nodes, 
+what they do, and how we use them in Taipy pipelines.
+
+Data nodes in Taipy are like tools to work with data. They don't hold data themselves, 
+but they know how to get it from different places. 
+Think of a data node as something that can read and write data, and it's really good at doing that.
+
+Now, we'll talk about two types of data nodes:
+
+- **Input Data Nodes**: These are data nodes that help us bring data into our system.
+
+- **Output Data Nodes**: These are data nodes that help us send data out of our system.
+
+So, data nodes are like helpers for handling data, 
+and they come in these two varieties: input and output.
+
+![Data Nodes](data_notes_2.svg){width=100%}
+
+Taipy has a set of predefined data nodes ready to be used when configuring your pipelines. 
+
+Here’s the list of predefined data nodes:
+
+![Data Nodes](data_notes.png){width=70%}
+
+## Pickle Data Node
+
+The Pickle data node is the standard data node in Taipy. 
+It can handle various types of Python stuff like strings, numbers, lists, dictionaries, 
+models (for machine learning or other things), and data tables. Here's some code that uses 
+two Pickle data nodes: one for getting data in and one for sending data out.
+
+- model is an input *Pickle* data node. It looks at a Pickle file called *model.p* and gets data from there.
+
+- predictions is an output data node, but right now, it doesn't have any data in it. 
+  We haven't told it where to get data from yet.
+
+<video width="640" height="360" controls>
```

Comment:
Use some style, host the video

Reply:
Do you mean to add a title name to the video or?